packages/forms/src/styled/common/StyledField.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'forms.field';
+
+export const StyledField = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  direction: ${props => props.theme.rtl && 'rtl'};
+  /* as <fieldset> resets */
+  margin: 0;
```

Comment:
Since this is locked as a `div` do we need these resets from the CSS side?

Reply:
It is valid (and perhaps desirable – see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset#example – in limited cases) to use `<Field as="fieldset">`. It's trivial for us to keep these and style accordingly.